examples/todos/README.md

```diff
@@ -1,3 +1,13 @@
 ## Test Vue.js Examples
 
 This suite tests the examples that ship with the open source Vue.js project.
+
+### Running Journeys via Synthetics CLI
+
+```
+// Install the dependencies
+npm install
+
+// Invoke the Synthetics runner
+./node_modules/.bin/elastic-synthetics .
```

Comment:
```suggestion
npx elastic-synthetics .
```

should be equivalent (I think I can change the heartbeat logic here too). See: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b